### PR TITLE
Cmake

### PR DESCRIPTION
**Pull request by mattn**

### CODE REVIEW

**Comment by mattn** [comment]
Tests passed YAY. cc: @objectx
Can I use your commits?

File: example/json/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+cmake_minimum_required (VERSION 3.15)
+
+set (t_ example-json)
+    add_executable (${t_} main.cxx)
+    target_link_libraries (${t_} PRIVATE clask-core)
+    target_include_directories(${t_} PUBLIC .)
+    file(COPY ${CMAKE_SOURCE_DIR}/example/json/public DESTINATION ${CMAKE_BINARY_DIR}/example/json)
```
Comment (objectx) [inline example/json/CMakeLists.txt]: This line could be:
```CMake
file (COPY ${CMAKE_CURRENT_SOURCE_DIR}/public DESTINATION ${CMAKE_CURRENT_BINARY_DIR})
```

**Comment by objectx** [comment]
@mattn Thank you for using my half-baked commit!

File: example/json/CMakeLists.txt (hunk 1 above)
Comment (mattn) [inline example/json/CMakeLists.txt]:
```suggestion
    file (COPY ${CMAKE_CURRENT_SOURCE_DIR}/public DESTINATION ${CMAKE_CURRENT_BINARY_DIR})
```

Comment (mattn) [reply example/json/CMakeLists.txt]: Thank you.

**Pull request merged**